client/components/parts/SignUp.jsx

```diff
@@ -34,7 +34,7 @@ export default class SignUp extends PureComponent {
               <SignUpForm />
             </div>
             <div className="modal-footer">
-              <Link className="btn btn-primary" to="/editor">Create Site</Link>
+              <Link className="btn btn-primary" to="/editor/create">Create Site</Link>
```

Comment:
Will need to confirm with @Inschato @JoshDPearson and @BrendonEarl depending on whether they were able to put a default JSON object in the DB so that we can retrieve it with 'create' or whatever else.